.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Bug Report
+about: Report a bug in Alpine.js
+title: ''
+labels: bug
```

Comment:
I don't know that we need to pre-label as bug, a lot of issues end up being enhancements/won't do's